### ISSUE TITLE
Version strings are combined using a culture dependent NumberDecimalSeparator

### ISSUE THREAD

**Issue by glienard** [tjscience/RoboSharp#92]
Ran into this problem on one of our European servers.
```
System.FormatException: Input string was not in a correct format.
  at System.Number.ParseDouble(String value, NumberStyles options, NumberFormatInfo numfmt)
  at System.Convert.ToDouble(String value)
  at RoboSharp.VersionManager.get_Version() in C:\GitHub\RoboSharp\RoboSharp\VersionManager.cs:line 41
  at RoboSharp.CopyOptions..ctor() in C:\GitHub\RoboSharp\RoboSharp\CopyOptions.cs:line 52
  at RoboSharp.RoboCommand..ctor() in C:\GitHub\RoboSharp\RoboSharp\RoboCommand.cs:line 24
```

**Comment by Jascha-W3DG** [comment]
Any chance the fix will make it into the stable release? This issue also happens on French systems.

**Comment by Jascha-W3DG** [comment]
Hey,

sadly this was not the best fix, since its now broken on German systems too

**Comment by tjscience** [comment]
@Jascha-W3DG, this has been fixed and 1.2.4 has been pushed to nuget.
